src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -2,10 +2,21 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
 
+@Entity
+@Table(name = "movies")
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String title;
+    @OneToMany
     private List<Actor> actors;
```

Comment:
Does your tests pass with default fetch type?

Reply:
![image](https://user-images.githubusercontent.com/110828777/203874700-e3733718-72b4-410a-8026-307de370d743.png)

Reply:
Excactly! EAGER is a default type for *ToOne but you have *ToMany

Reply:
let's add a joinColumn annotation here too.
like

```
@JoinTable(name = "movies_actors",
            joinColumns = @JoinColumn(name = "movie_id"),
            inverseJoinColumns = @JoinColumn(name = "actor_id"))
```